cast/app.css

```diff
@@ -77,15 +77,20 @@ html {
   text-align: center;
   margin-left: 2rem;
   display: none;
-  border-radius: 0.25rem;
+  cursor: pointer;
+}
+
+a {
```

Comment:
Didn't understand the need for this?

Reply:
To make it look more like a button and less like a hyperlink